src/frontend/sharedComponents/ActionTab/ItemButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {FC} from 'react';
+import {ActionTabItems} from './ActionTab';
+import {Pressable, StyleSheet, Text, View} from 'react-native';
+import {CustomCircleIcon} from './CustomCircleIcon';
+
+export const ItemButton: FC<ActionTabItems> = ({onPress, icon, label}) => (
+  <Pressable onPress={onPress} style={styles.itemContainer}>
```

Comment:
Can you change to the a `TouchableOpacity` please?

Reply:
okay, no problem :)